hack/release/stable-pr.sh

```diff
@@ -12,6 +12,8 @@ if [[ $(releaseType $GIT_TAG) != $RELEASE_TYPE_STABLE ]]; then
   exit 1
 fi
 
+updateKarpenterCoreGoMod $GIT_TAG
```

Comment:
This couples the karpenter-core upgrade with the stable release PR, but there are cases where we don't want to upgrade to the latest karpenter-core yet. For example, if a provisioner spec change goes into karpenter-core, but the functionality hasn't been added to aws karpenter yet. 

I think this needs to be in a separate PR minimally so that we don't hold the chart update from getting merged if we're not ready to merge karpenter-core yet.

Reply:
ahh ok, I missed the RELEASE_VERSION bit